.goreleaser.yaml

```diff
@@ -61,3 +61,10 @@ brews:
     commit_author:
       name: WorkOS
       email: support@workos.com
+dockers:
+  - ids:
+      - linux
```

Comment:
I wonder if it's worth building/publishing images for `x86-64` and `ARM`? (For Windows and ARM machines)

Reply:
Ok, in that case, we might need to update the tags in the ECR public gallery because it shows Windows and ARM64 as supported platforms
![Screenshot 2024-08-21 at 3 47 59 PM](https://github.com/user-attachments/assets/0ea14574-24b3-4ccc-8e7c-9471f8fa2953)